Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean

```diff
@@ -573,6 +573,10 @@ open Module.Free
 
 open scoped nonZeroDivisors
 
+/-- The image of the ring of integers of `K` in the mixed space. -/
+protected abbrev integerLattice : Submodule ℤ (mixedSpace K) :=
+  LinearMap.range ((mixedEmbedding K).comp (algebraMap (𝓞 K) K)).toIntAlgHom.toLinearMap
```

Comment:
```suggestion
  ((mixedEmbedding K).comp (algebraMap (𝓞 K) K)).toIntAlgHom.toLinearMap.range
```
Does this work? I am afraid it doesn't, but it would be nice.

Reply:
No, it doesn't work. Somehow `LinearMap.range` never works...

Reply:
(I mean with the dot notation)

Reply:
Because it doesn't take a `LinearMap`, but a term of a type with an instance of `LinearMapClass` (I think).